src/praisonai/praisonai/cli/commands/agent.py

```
@@ -0,0 +1,115 @@
+
+from typing import Optional
+
+import typer
+
+from ..output.console import get_output_controller
+
+app = typer.Typer(help="Manage custom agents")
+
+
+@app.command()
+def list(
```

Ruff (0.15.17)

[error] Variable `list` is shadowing a Python builtin

(A001)